pgsql_big_dedupe_example/pgsql_big_dedupe_example.py

```diff
@@ -195,9 +188,9 @@
 print 'creating inverted index'
 
 for field in deduper.blocker.index_fields:
-    c2 = con2.cursor('c2')
+    c2 = con.cursor('c2')
     c2.execute("SELECT DISTINCT %s FROM processed_donors" % field)
-    field_data = (row for row in c2)
+    field_data = set(row[field] for row in c2)
```

Comment:
We expect it to be an iterable of unique elements.

On Mon, Aug 17, 2015 at 2:56 PM Justin Manley <notifications@github.com>
wrote:

> In pgsql_big_dedupe_example/pgsql_big_dedupe_example.py
> <https://github.com/datamade/dedupe-examples/pull/25#discussion_r37229528>
> :
>
> >      c2.execute("SELECT DISTINCT %s FROM processed_donors" % field)
> > -    field_data = (row for row in c2)
> > +    field_data = set(row[field] for row in c2)
>
> Even though the documentation specifies that blocker.index expects its
> first argument to be a value of type set
> <http://dedupe.readthedocs.org/en/latest/API-documentation.html#Dedupe.blocker.index>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datamade/dedupe-examples/pull/25/files#r37229528>.
>